src/bentoml_cli/bentos.py

```diff
@@ -47,298 +59,288 @@ def parse_delete_targets_argument_callback(
     return delete_targets
 
 
-def add_bento_management_commands(cli: Group):
-    import bentoml
-    from bentoml import Tag
-    from bentoml.bentos import import_bento
-    from bentoml._internal.utils import rich_console as console
-    from bentoml._internal.utils import calc_dir_size
-    from bentoml._internal.utils import human_readable_size
-    from bentoml._internal.utils import resolve_user_filepath
-    from bentoml._internal.bento.bento import Bento
-    from bentoml._internal.bento.bento import DEFAULT_BENTO_BUILD_FILE
-    from bentoml._internal.configuration import get_quiet_mode
-    from bentoml._internal.bento.build_config import BentoBuildConfig
-    from bentoml._internal.configuration.containers import BentoMLContainer
-
-    bento_store = BentoMLContainer.bento_store.get()
-    yatai_client = BentoMLContainer.yatai_client.get()
-
-    @cli.command()
-    @click.argument("bento_tag", type=click.STRING)
-    @click.option(
-        "-o",
-        "--output",
-        type=click.Choice(["json", "yaml", "path"]),
-        default="yaml",
-    )
-    def get(bento_tag: str, output: str) -> None:  # type: ignore (not accessed)
-        """Print Bento details by providing the bento_tag.
-
-        \b
-        bentoml get iris_classifier:qojf5xauugwqtgxi
-        bentoml get iris_classifier:qojf5xauugwqtgxi --output=json
-        """
-        bento = bento_store.get(bento_tag)
-
-        if output == "path":
-            console.print(bento.path)
-        elif output == "json":
-            info = json.dumps(bento.info.to_dict(), indent=2, default=str)
-            console.print_json(info)
-        else:
-            info = yaml.dump(bento.info, indent=2, sort_keys=False)
-            console.print(Syntax(info, "yaml", background_color="default"))
-
-    @cli.command(name="list")
-    @click.argument("bento_name", type=click.STRING, required=False)
-    @click.option(
-        "-o",
-        "--output",
-        type=click.Choice(["json", "yaml", "table"]),
-        default="table",
-    )
-    def list_bentos(bento_name: str, output: str) -> None:  # type: ignore (not accessed)
-        """List Bentos in local store
-
-        \b
-        # show all bentos saved
-        $ bentoml list
-
-        \b
-        # show all verions of bento with the name FraudDetector
-        $ bentoml list FraudDetector
-        """
-        bentos = bento_store.list(bento_name)
-        res = [
-            {
-                "tag": str(bento.tag),
-                "size": human_readable_size(calc_dir_size(bento.path)),
-                "creation_time": bento.info.creation_time.astimezone().strftime(
-                    "%Y-%m-%d %H:%M:%S"
-                ),
-            }
-            for bento in sorted(
-                bentos, key=lambda x: x.info.creation_time, reverse=True
+bento_store = BentoMLContainer.bento_store.get()
+yatai_client = BentoMLContainer.yatai_client.get()
+
+@click.group('bentos')
+def bento_commands():
+    pass
```

Comment:
We can add a quick docstring for this to show it on the CLI

Reply:
👍 should do this.